pkg/sql/schemachanger/eav/eavquery/testdata/basic

```diff
@@ -0,0 +1,57 @@
+schema
```

Comment:
It's mildly confusing that there are 2 different test directories for these datadriven tests - why not put in 1 big one? Are we anticipating a ton of these kinds of tests?